pkgs/development/compilers/miranda/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, lib, fetchurl, gcc }:
+
+stdenv.mkDerivation rec {
+  version = "2.066";
+  pname = "miranda";
+
+  src = fetchurl {
+    url = "https://www.cs.kent.ac.uk/people/staff/dat/miranda/src/mira-${builtins.replaceStrings [ "." ] [ "" ] version}-src.tgz";
+    sha256 = "1sqn031agwwgq3pka55pxd6m22dagmkxjhbwrni8gpigihg2h72j";
+  };
+
+  enableParallelBuilding = true;
+
+  configurePhase = ''
+    patchShebangs quotehostinfo
```

Comment:
Could you move this to the other phase in the bottom, see https://discourse.nixos.org/t/document-attribute-ordering-in-package-expressions/4887.